app/assets/stylesheets/application.scss

```diff
@@ -129,6 +129,10 @@ h3,h4{
   }
 }
 
+h4 a{
```

Comment:
This is a bit too broad, suspect it will affect other areas of the site where we use links in h4 headings